app/uk/gov/hmrc/helptosavereminder/config/AppConfig.scala

```diff
@@ -17,16 +17,30 @@
 package uk.gov.hmrc.helptosavereminder.config
 
 import javax.inject.{Inject, Singleton}
-import play.api.Configuration
+import play.api.{Configuration, Environment, Mode}
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
+class AppConfig @Inject()(val config: Configuration, val environment: Environment, val servicesConfig: ServicesConfig) {
```

Comment:
I do not think that environment is used any more

Reply:
Done